libflux/src/flux/formatter/mod.rs

```diff
@@ -1,32 +1,57 @@
 #![allow(missing_docs)]
-use crate::ast::{self, walk::Node};
+use crate::ast::{self, walk::Node, File};
+use crate::parser::parse_string;
 use crate::Error;
 
 use chrono::SecondsFormat;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub fn format_from_js_file(js_file: JsValue) -> String {
+    if let Ok(file) = js_file.into_serde::<File>() {
+        if let Ok(converted) = convert_to_string(&file) {
+            return converted;
+        }
+    }
+    "".to_string()
+}
+
+pub fn convert_to_string(file: &File) -> Result<String, String> {
+    let mut formatter = Formatter::default();
+    formatter.format_file(file, true);
+    formatter.output()
+}
+
+pub fn format(contents: String) -> Result<String, String> {
```

Comment:
This method appears to only be used in the test cases. Should we move it to the test module?

Reply:
its meant to be the public interface when we start using the formatter in other places, just to encapsulate the creation of a formatter and executing said formatter